platform-includes/capture-error/flutter.mdx

```diff
@@ -1,4 +1,32 @@
 <PlatformContent includePath="capture-error/dart.mdx" />
 
-- Flutter-specific errors, such as using `FlutterError.onError`, are captured automatically
-- The SDK already runs your init `callback` on an error handler, such as [`runZonedGuarded`](https://api.flutter.dev/flutter/dart-async/runZonedGuarded.html) on Flutter versions prior to `3.3`, or [`PlatformDispatcher.onError`](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/onError.html) on Flutter versions 3.3 and higher, so that errors are automatically captured.
+### FlutterError.onError
+
+Flutter-specific errors, such as using `FlutterError.onError`, are captured automatically.
+
+### PlatformDispatcher.onError / runZonedGuarded
+
+The SDK already runs your init `callback` on an error handler, such as [`runZonedGuarded`](https://api.flutter.dev/flutter/dart-async/runZonedGuarded.html) on Flutter versions prior to `3.3`, or [`PlatformDispatcher.onError`](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/onError.html) on Flutter versions 3.3 and higher, so that errors are automatically captured.
```

Comment:
doesn't the runZonedGuarded only happen on Web?

Reply:
Afaik there was no onError callback on version with Flutter < 3.3, so we'd also use it there.